### DISCUSSION TITLE
what is the nerd font name you choose?

### DISCUSSION THREAD

**Discussion by yushaku** [LazyVim/LazyVim#450] [Q&A] [answered]
currently, i use dejavu nerd font but it does not support all icons. i have changed several but the icons do not display.
![image](https://user-images.githubusercontent.com/72312124/225992985-3cae248a-5ed6-41df-a50f-c310b8023e7e.png)

**Comment by folke** [comment]
Any nerd font is fine, as long as it's not too old. Lazyvim is compatible with the stable nerd font releases

**Comment by al3jandr0** [comment, marked as answer]
@yushaku you may need to refresh the _font cache_ after installing a new font: `sudo fc-cache -fr`

See this tutorial for terminal-specific troubleshooting of fonts
https://www.youtube.com/watch?v=mQdB_kHyZn8
